### DISCUSSION TITLE
Problem with Flexget not getting Title info from Magnet, folder/files ends up the name of torrent

### DISCUSSION THREAD

**Discussion by paulpoco** [Flexget/Flexget#2815] [Q&A]
Problem with Flexget not getting Title info from Magnet that fails, folder/files ends up the name of torrent.  And passes the "content_filename" onto Deluge.

It seems worse if I do it like this:
![Screenshot_20210121-081505_TeamViewer](https://user-images.githubusercontent.com/801414/105379341-94e84d80-5bc1-11eb-82ad-5d954cb912e3.jpg)

Vs like this but I really don't want the extra folder for each episode:
![Screenshot_20210121-081301_TeamViewer](https://user-images.githubusercontent.com/801414/105379415-a7628700-5bc1-11eb-8ad1-a5edd851fcea.jpg)

**Comment by gazpachoking** [comment]
Hmm. First thing I'd probably do is to move all of your settings from the `set` plugin into their actual plugins. (`quality` and `upgrade` to series plugin, and the rest to deluge plugin)
After that, I'm not quite sure what your question is? When the metadata for the torrent doesn't download in time, there isn't much flexget can do about setting the content_filename.

**Reply by paulpoco** [reply]
Ok I'll try your suggestions 1st.  Thanks.

**Comment by frankyw** [comment]
@paulpoco you may want to look at something like [Filebot](https://www.filebot.net/) to process files after download,  it can be called via CLI by Deluge after completion. It is designed specifically for media file renaming.

**Reply by ksurl** [reply]
Flexget can handle renaming in a separate task if you use local or disable seen. No need for external tools
